### DISCUSSION TITLE
[Question]: Error using Claude 3 model

### DISCUSSION THREAD

**Discussion by QYQSDTC** [danny-avila/LibreChat#1988] [Troubleshooting] [answered]
### What is your question?

When I chatted with Claude-3-opus-20240229 model, I got the error message below:
![CleanShot-GpyReDnE@2x](https://github.com/danny-avila/LibreChat/assets/15608306/417d52c2-5cd8-4459-b560-feca65938a8b)
I am using a reverse proxy, I checked their documentation it seems that the URL should be `/v1/completions` rather than `/v1/compete`, how can I change the POST URL from `/v1/complete` to `/v1/completions`?

### More Details

The reverse proxy I am using https://oneapi.gptnb.me/about

### What is the main subject of your question?

Endpoints

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by danny-avila** [comment, marked as answer]
The Anthropic endpoint uses the official Anthropic API, so it is going to be quite different from oneapi. [Anthropic API docs.](https://docs.anthropic.com/claude/reference/messages_post)

You can use the oneapi proxy as a Custom endpoint, though: https://docs.librechat.ai/install/configuration/custom_config.html

**Reply by acerspyro** [reply]
Thank you, my understanding is that since Claude 3, Anthropic is now using their own API rather than OneAPI?

Would this require changes to LibreChat to support Claude 3 out of the box?

(I'm not using a reverse proxy, but I have the same error on latest main)

EDIT: I'm actually getting a different error, my bad. Not sure if it's related.

![image](https://github.com/danny-avila/LibreChat/assets/4400999/53ce2991-1c5a-4940-8049-dbd50f67ccab)

**Reply by fuegovic** [reply]
@acerspyro 
LibreChat has already been updated to support Claude 3: https://github.com/danny-avila/LibreChat/pull/1984

**Reply by acerspyro** [reply]
Indeed, I upgraded to latest `main` and got this error. Updated my .env and did a `docker-compose down` and `docker-compose up -d`. The only Anthropic config I have is the API key in my .env. Anything else I'm missing?

**Reply by fuegovic** [reply]
Try to stop the container, remove the librechat service, clean up cached and dangling images, and re-pull the updated librechat-dev image

**Reply by acerspyro** [reply]
Thank you, it works! I had already removed my contains and pruned my images, but for some reason it hadn't pruned librechat-dev. Code 18, the joys of Docker :)

I'm done hijacking this person's thread 😅

**Reply by QYQSDTC** [reply]
I gave up, I am now using the official API key 😆
